net/mwan3/files/usr/sbin/mwan3track

```diff
@@ -128,6 +128,11 @@ main() {
 		echo "$(get_uptime)" > /var/run/mwan3track/$1/ONLINE
 		env -i ACTION="connected" INTERFACE="$1" DEVICE="$2" /sbin/hotplug-call iface
 	fi
+
+	# Determ kernel version to fix routing issue for tunnel interfaces
+	FIX_KERNEL_ISSUE="no"
+	opkg compare-versions $(uname -r) '<=' '4.14.209' && FIX_KERNEL_ISSUE="yes"
```

Comment:
Does '<=' work here? This comment suggested it was buggy: https://github.com/openwrt/packages/commit/6bd3f5c3777c5cbad4ac9156fa5b0fda26af14a9#commitcomment-45865624